conans/client/installer.py

```diff
@@ -129,28 +129,10 @@ def _build_node(self, conan_ref, conan_file, build_mode):
 
         self._handle_system_requirements(conan_ref, package_reference, conan_file, output)
 
-        # Check if package is corrupted
-        valid_package_digest = self._paths.valid_package_digest(package_reference)
```

Comment:
Functionality moved to ConanProxy, as it is very similar to the one of get_conanfile()